src/main/webapp/DastFreeStyle.js

```diff
@@ -0,0 +1,1199 @@
+    }
+
+    setPostmanSettings(postmanSettings) {
+        jq('#apiTypeList').val('postman');
+        this.onApiTypeChanged();
+    }
+
+    setSelectedEntitlementValue(entitlements) {
+
+        let currValSelected = false;
+        let curVal = getEntitlementDropdownValue(this.scanSettings.entitlementId, this.scanSettings.entitlementFrequencyType);
+        let entitlement = jq('#entitlementSelectList');
+        for (let ts of Object.keys(entitlements)) {
+            let at = entitlement[ts];
+            console.log(at.text);
+            console.log(at.value);
+            if (curVal !== undefined && at.value !== undefined && curVal.toLowerCase() === at.value.toLowerCase()) {
+                currValSelected = true;
+                entitlement.append(`<option value="${at.text}" selected>${at.text}</option>`);
+                jq('#entitlementId').val(at.text);
+            } else {
+                if(at.text !== null||undefined) {
+                    entitlement.append(`<option value="${at.text}">${at.text}</option>`);
+                }
+            }
+        }
+    }
+
+    setPatchUploadFileId() {
+
+        if (!Object.is(this.scanSettings.loginMacroFileId, null) &&
+            this.scanSettings.loginMacroFileId !== undefined && this.scanSettings.loginMacroFileId > 0) {
+            jq('#loginMacroId').val(this.scanSettings.loginMacroFileId);
+        }
+        if (!Object.is(this.scanSettings.workflowdrivenAssessment, null)
+            && this.scanSettings.workflowdrivenAssessment !== undefined) {
+
+            if (this.scanSettings.workflowdrivenAssessment.workflowDrivenMacro.fileId > 0)
+                jq('#workflowMacroId').val(this.scanSettings.workflowdrivenAssessment.workflowDrivenMacro.fileId);
+        }
+    }
+
+    setNetworkSettings() {
+
+        if (!Object.is(this.scanSettings.networkAuthenticationSettings, null)
+            && !Object.is(this.scanSettings.networkAuthenticationSettings, undefined)) {
+            jq('#networkUsernameRow').find('input').val(this.scanSettings.networkAuthenticationSettings.userName);
+            jq('#networkPasswordRow').find('input').val(this.scanSettings.networkAuthenticationSettings.password);
+            if (jq('#webSiteNetworkAuthSettingEnabledRow').find('input:checkbox:first').prop('checked') === false) {
+                jq('#webSiteNetworkAuthSettingEnabledRow').find('input:checkbox:first').trigger('click');
+            }
+            let np = jq('#networkPasswordRow').find('input');
+            np.attr('type', 'password');
+        }
+    }
+
+    resetNetworkSettings() {
+
+        jq('#networkUsernameRow').find('input').val(undefined);
+        jq('#networkPasswordRow').find('input').val(undefined);
+        jq('#ddlNetworkAuthType').prop('selectedIndex', 0);
+        jq('#webSiteNetworkAuthSettingEnabledRow').find('input:checkbox:first').trigger('click');
+    }
+
+    setScanType() {
+        if (this.scanSettings !== undefined && this.scanSettings !== null) {
```

Comment:
I'm perfectly fine with this way, it's more explicitly, but if you'd like to clean these if statements up by using truthiness it's perfectly safe to do...

if (this.scanSettings)